.connector-store/meta.json

```diff
@@ -0,0 +1,77 @@
+{
+    "name": "HTTP",
+    "owner": "WSO2",
+    "product": "MI",
+    "category": "Networking",
```

Comment:
Shall we stick with a category that is already mentioned here? https://store.wso2.com/?page=1&product=Micro+Integrator
Developer Tools would be more suitable.